javascript/react/basics-i/rendering-multiple-components.md

```diff
@@ -110,4 +111,9 @@ What is the name of the **identity attribute** that must be included when creati
 * `identity`
 * `ref`
 
+---
+## Footnotes
+[1:keys]
+To get a better understanding of this, let's take a look at how React decides where it should re-render a component. In its rawest form, a React component is just an object with a set of properties. Each time a change occurs in these properties, React will compare the new and old versions and, if a difference is detected, it will redraw the component. It's important to note that any property of a React element can change, we could make an `input` into a `button`, add/remove and `id` from a `form`, etc. This is why, when dealing with multiple elements at the same level, a unique `key` is needed so that React knows how to keep track of the order of elements no matter how the elements themselves change. 
```

Comment:
```suggestion
To get a better understanding of this, let's take a look at how React decides where it should re-render a component. In its rawest form, a React component is just an object with a set of properties. Each time a change occurs in these properties, React will compare the new and old versions and, if a difference is detected, it will redraw the component. It's important to note that any property of a React element can change, we could make an `input` into a `button`, add/remove an `id` from a `form`, etc. This is why, when dealing with multiple elements at the same level, a unique `key` is needed so that React knows how to keep track of the order of elements no matter how the elements themselves change. 
```